.github/workflows/component-library-tag-design.yml

```
@@ -0,0 +1,30 @@
+name: Component-Library-Tag-Design
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, synchronize]
+    paths:
+      - 'frontend/packages/component-library-styles/**'
+      - 'frontend/packages/component-library/**'
+      - 'frontend/packages/fonts/**'
+
+jobs:
+  request:
+    name: Request review from design team on opened Component Library PRs
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

Infrastructure had requested we use the same Ubuntu version

---

not trying to thrash here, but for tooling I'm comfortable with `latest`, whereas when testing our building code headed for production, we should use a consistently pinned version like `24.04`